astro/src/content/quickstarts/quickstart-javascript-react-web.mdx

```diff
@@ -0,0 +1,198 @@
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/pages/AccountPage.js"
+            lang="jsx"/>
+
+#### Create A Make Change Page
+
+The make change page will be a simple page that displays an input field for the user to enter a dollar amount and a button to convert that amount into coins. Create a new file `src/pages/MakeChangePage.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/pages/MakeChangePage.js"
+            lang="jsx"/>
+
+## Authentication
+
+You now have created a basic React application with a home page and an account page.
+
+Depending on the user's authentication state, the login or logout button should be displayed in the header. For this create a new directory `components` in `src` and a new file `src/components/LogoHeader.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/components/LogoHeader.js"
+            lang="jsx"/>
+
+Additionally, we want to display different menu items. For this create a new file `src/components/MenuBar.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/components/MenuBar.js"
+            lang="jsx"/>
+
+The next step is to tie it all together. Update the `src/App.js` file to add the routes and the header:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/App.js"
+            lang="jsx"/>
+
+## Running the Application
+
+You can now run the application with the following command:
+
+```shell
+npm start
+```
+
+You can now open up an incognito window and navigate to http://localhost:3000. You will be greeted with the home page. Log in with the user account you created when setting up FusionAuth, and you'll be redirected to the account page.
+
+<Aside type="note">
+    The username and password of the `example user` can be found in the <a href="#fusionauthconfig">FusionAuth via Docker</a> section at the top of this article.
+</Aside>
+
+## Next Steps
+This quickstart is a great way to get a proof of concept up and running quickly, but to run your application in production, there are some things you're going to want to do.
+
+### FusionAuth Customization
+FusionAuth gives you the ability to customize just about everything with the user's experience and your application's integration. This includes
+* [Hosted pages](/docs/v1/tech/themes/) such as login, registration, email verification, and many more
+* [Email templates](/docs/v1/tech/email-templates/email-templates)
+* [User data and custom claims in access token JWTs](/articles/tokens/jwt-components-explained)
```

Comment:
This should point to the fusionauth populate lambda jwt.

Reply:
Do you mean changing: `/articles/tokens/jwt-components-explained` to `/docs/v1/tech/lambdas/jwt-populate`?
It got copied from the ruby quickstart, maybe we need to change it there too then.
@mooreds

Reply:
Good point. Let's leave it be.